shared/clinical-code-sets/medications/other-anxiolytics-and-hypnotics/1/other-anxiolytics-and-hypnotics.readv2.txt

```diff
@@ -1,17 +1,69 @@
-d1gz.00	ZALEPLON 10mg capsules
-d1gy.00	ZALEPLON 5mg capsules
-d1g2.00	SONATA 10mg capsules
-d1g1.00	SONATA 5mg capsules
\ No newline at end of file
+d1c..00	TRICLOFOS SODIUM
+d1c1.00	*TRICLOFOS 500mg/5mL liquid
+d19..00	PROMETHAZINE HCL [HYPNOTIC] see section c8i..
+d1h..00	MELATONIN
+d1hz.00	MELATONIN 2mg m/r tablets
+d13..00	*DICHLORALPHENAZONE
+d13z.00	DICHLORALPHENAZONE 225mg/5mL elixir
+d13y.00	DICHLORALPHENAZONE 650mg tablets
+d12..00	CLOMETHIAZOLE EDISYLATE [HYPNOTIC]
+d12w.00	CLOMETHIAZOLE EDISILATE 250mg/5mL syrup
+d12w.12	CLOMETHIAZOLE(base) 31.5mg/mL syrup
+d12v.00	CLOMETHIAZOLE(base) 192mg capsules
+d11..00	CHLORAL HYDRATE
+d119.00	CHLORAL HYDRATE 500mg/5mL syrup
+d118.00	CHLORAL HYDRATE 143mg/5mL elixir
+d114.00	CHLORAL HYDRATE 500mg capsules
+d112.00	CHLORAL PAEDIATRIC 200mg/5mL elixir
+d111.00	*CHLORAL 500mg/5mL mixture
+d117.00	CHLORAL HYDRATE 414mg tablets
+d117.11	CHLORAL BETAINE 707mg tablets
+c8i..00	PROMETHAZINE HYDROCHLORIDE [ANTIHISTAMINE]
+c8iz.00	PROMETHAZINE 50mg/2mL injection
+c8iy.00	PROMETHAZINE 25mg/1mL injection
+c8ix.00	PROMETHAZINE 5mg/5mL elixir
+c8iw.00	PROMETHAZINE 25mg tablets
+c8iv.00	PROMETHAZINE 10mg tablets
+c8iu.00	PROMETHAZINE 20mg tablets
+c8a..00	HYDROXYZINE HYDROCHLORIDE [ANTIHISTAMINE]
+c88..00	DIPHENHYDRAMINE HYDROCHLORIDE [ANTIHISTAMINE]
+c88z.00	DIPHENHYDRAMINE HYDROCHLORIDE 25mg capsules
+c88y.00	DIPHENHYDRAMINE HYDROCHLORIDE 50mg capsules
+c88x.00	DIPHENHYDRAMINE HYDROCHLORIDE 6.25mg/5mL syrup
+c887.00	DIPHENHYDRAMINE HYDROCHLORIDE 50mg tablets
+c884.00	DIPHENHYDRAMINE HYDROCHLORIDE 10mg/5mL syrup
+c883.00	DIPHENHYDRAMINE HCL 25mg tablets
+c8k..00	ALIMEMAZINE TARTRATE [ANTIHISTAMINE]
+c8kz.00	ALIMEMAZINE 30mg/5mL syrup
+c8ky.00	ALIMEMAZINE 7.5mg/5mL syrup
+c8kx.00	ALIMEMAZINE 10mg tablets
+d2c..00	MEPROBAMATE
+d2c2.00	MEPROBAMATE 400mg tablets
+d2c1.00	*MEPROBAMATE 200mg tablets
+d28..00	HYDROXYZINE HCL [ANXIOLYTIC]
+d28z.00	HYDROXYZINE HCL 10mg/5mL syrup
+d28y.00	HYDROXYZINE HCL 25mg tablets
+d28x.00	HYDROXYZINE HCL 10mg tablets
+d25..00	CHLORMEZANONE
+d25z.00	*CHLORMEZANONE 200mg tablets
+d2f..00	BUSPIRONE HYDROCHLORIDE
+d2f5.00	BUSPIRONE 10mg tablets
+d2f1.00	BUSPIRONE 5mg tablets
+dhg..00	PROMETHAZINE THEOCLATE
+dhgz.00	PROMETHAZINE THEOCLATE 25mg tablets
+dhf..00	PROMETHAZINE [nausea] [see c8i..]
+dh6..00	DIPHENHYDRAMINE HYDROCHLORIDE [NAUSEA]
+o59..00	PROMETHAZINE HCL [ANAESTHESIA]
+o51..00	CLOMETHIAZOLE EDISYLATE [ANAESTHESIA]
+o51z.00	CLOMETHIAZOLE EDISYLATE 8mg/mL intravenous infusion
+do3..00	CLOMETHIAZOLE EDISYLATE [CENTRAL NERVOUS SYSTEM USE]
+do3z.00	CLOMETHIAZOLE EDISYLATE 8mg/mL intravenous infusion
+dibT.00	PARACETAMOL+DIPHENHYDRAMINE HYDROCHLORIDE 120mg/12.5mg/5mL sugar free oral solution
+dibO.00	PARACETAMOL+DIPHENHYDRAMINE HYDROCHLORIDE 500mg/25mg tablets
+d3...00	BARBITURATES/METHYPRYLONE
+d32..00	*METHYPRYLONE
+dC1..00	SODIUM OXYBATE
+dC1z.00	SODIUM OXYBATE 500mg/mL oral solution
+m347.00	DIPHENHYDRAMINE HYDROCHLORIDE 2% cream
+9Ngt.00	On melatonin for sleep disorder
```

Comment:
This isn't a medication code - but it does indicate that the person is taking the drug. The problem is that this code would not appear in the GP_Medications table (would be in GP_Events), so you'd need to remember to search both tables when using this code set. In the past for e.g. flu vaccine codes where there are medication codes (the actual vaccine), and procedure codes (the vaccination event) which are both used, I have created a "flu-vaccine" code set under medications, and a "flu-vaccination" code set under procedures. You can then search GP_Medications for the first, and GP_Events for the second. Having said all that, in this instance I suspect you can remove the 9Ngt code from this list as there will be no-one on the drug who has this code, but doesn't have a medication code.